app/components/task-details.js

```diff
@@ -107,19 +107,13 @@ export default Component.extend({
 
       @method saveTaskBody
      */
-    saveTaskBody() {
+    applyEditTask() {
       let component = this;
```

Comment:
I know it is out of the scope of this PR, but is this still necessary? The arrow function below allows us to keep the lexical `this`. I think we can delete this and update line 115 to be `this.set('isEditingBody', false)` and reduce some redundancy.